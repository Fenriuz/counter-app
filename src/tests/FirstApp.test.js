import FirstApp from '../FirstApp';
import { shallow } from 'enzyme';

describe('Tests for <FirstApp />', () => {
  //test('Show the message "Hello, I am Alex"', () => {
  //const greeting = 'Hello, I am Alex';
  //const { getByText } = render(<FirstApp greeting={greeting} />);
  //expect(getByText(greeting)).toBeInTheDocument();
  //});

  test('Should show <FirstApp /> correctly', () => {
    const greeting = 'Hello, I am Alex';
    const wrapper = shallow(<FirstApp greeting={greeting} />);

    expect(wrapper).toMatchSnapshot();
  });

  test('Should show the subtitle sent by the props', () => {
    const greeting = 'Hello, I am Alex';
    const subtitle = 'I am a subtitle';
    const wrapper = shallow(
      <FirstApp greeting={greeting} subtitle={subtitle} />
    );

    const textParagraph = wrapper.find('p').text();
    expect(textParagraph).toBe(subtitle);
  });
});
