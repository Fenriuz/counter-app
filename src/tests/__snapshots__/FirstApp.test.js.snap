// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests for <FirstApp /> Should show <FirstApp /> correctly 1`] = `
<Fragment>
  <h1>
    Hello, I am Alex
  </h1>
  <p>
    I am a subtitle
  </p>
</Fragment>
`;
